core/arch/aarch64/instr_create.h

```diff
@@ -277,6 +277,16 @@
     opnd_add_flags((sht), DR_OPND_IS_SHIFT), (sha))
 #define INSTR_CREATE_b(dc, pc) \
   instr_create_0dst_1src((dc), OP_b, (pc))
+
+/**
+ * This INSTR_CREATE_xxx macro creates an instr_t with opcode OP_xxx and
+ * the given explicit operands, automatically supplying any implicit operands.
```

Comment:
Sure, I picked a doc string from some X86 instruction. But I agree it makes more sense to be specific here. I'll update the comment. 

Missing docs for the other macros is an issue that needs to be addressed, but we should try to include proper doc strings for  newly added instructions.